packages/nimbus/src/components/icon-button/icon-button.mdx

```diff
@@ -14,85 +14,644 @@ figmaLink: >-
+        Revert
+      </Button>
+    </Stack>
+  )
+}
+```
+
+> [!TIP]\
+> **Do**
+>
+> - When offering a toggle experience, connect these icon buttons
+> - Include a tooltip explaining the options.
+> - Selected option is using the “Solid” state.
+
+```jsx-live
+const App = () => {
+  return (
+    <ToggleButtonGroup.Root tone="primary" selectedKeys={["left"]}>
+      <ToggleButtonGroup.Button id="left">
+        <Icons.AccountTree />
+      </ToggleButtonGroup.Button>
+      <ToggleButtonGroup.Button id="right">
+        <Icons.Search />
+      </ToggleButtonGroup.Button>
+    </ToggleButtonGroup.Root>
+  )
+}
+```
+
+> [!CAUTION]\
+> **Don't**
+>
+> - If offering a toggle like option view, do not show options as separate
+> - Does not include tooltips or labels, makes it confusing to users to
+>   understand this is a either/or choice.
+
+```jsx-live
+const App = () => {
+  return (
+    <Stack direction="horizontal" gap="100">
+      <IconButton tone="primary" variant="solid">
+        <Icons.AccountTree />
+      </IconButton>
+      <IconButton tone="primary" variant="outline">
+        <Icons.Search />
+      </IconButton>
     </Stack>
   )
 }
 ```
 
-### Tones
+> [!TIP]\
+> **Do**
+>
+> - When pairing icon buttons together, use the ghost style.
+
+```jsx-live
+const App = () => {
+  return (
+    <Stack direction="horizontal" gap="100">
+      <IconButton tone="primary" variant="ghost">
+        <Icons.Edit />
+      </IconButton>
+      <IconButton tone="primary" variant="ghost">
+        <Icons.Delete />
+      </IconButton>
+    </Stack>
+  )
+}
+```
 
-Available tones.
+> [!CAUTION]\
+> **Don't**
+>
+> - Avoid having two primary solid icon buttons in the same area.
 
 ```jsx-live
 const App = () => {
-  const variants = ["solid", "outline", "ghost", "link"];
-  const colors = ["primary", "error"];
+  return (
+    <Stack direction="horizontal" gap="100">
+      <IconButton tone="primary" variant="solid">
+        <Icons.Edit />
+      </IconButton>
+      <IconButton tone="primary" variant="solid">
+        <Icons.Delete />
+      </IconButton>
+    </Stack>
+  )
+}
+```
 
+### Tooltips
+
+Make sure to place tooltips in areas where they will be seen easily.
+
+> [!TIP]\
+> **Do**
+>
+> - Place tooltips so that it makes information easy to read.
+
+```jsx-live
+const App = () => {
   return (
-    <Stack>
-      {colors.map((color) => (
-        <Stack key={color} direction="horizontal">
-          {variants.map((variant) => (
-            <IconButton colorPalette={color} key={variant} variant={variant}>
-              <Icons.AutoFixHigh/>
-            </IconButton>
-          ))}
-        </Stack>
-      ))}
+    <Stack width="185px">
+      <TooltipTrigger delay={0} closeDelay={0} isOpen>
+        <IconButton variant="ghost" tone="primary" alignSelf="end">
+          <Icons.ViewColumn />
+        </IconButton>
+        <Tooltip placement="top" offset={-37} crossOffset={-100}>
+          Table settings
+        </Tooltip>
```

Comment:
These tooltip examples here and below deviate a bit from the spec, but the spirit of the design is intact. `placement="left"` is buggy (tooltip appeared 20,000px at the bottom of the page) so I had to manually alter these using the offset API. I wasted enough time here trying to figure this out, so if anyone has any ideas, I'm all ears.

Reply:
Sounds good! https://commercetools.atlassian.net/browse/FCT-1506